demos/ios/MASVS-AUTH/MASTG-DEMO-0042/run.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+frida -U -f org.owasp.mastestapp.MASTestApp-iOS -l ./script.js -o output.txt
```

Comment:
Maybe this can solve the issue:

```suggestion
frida -U -f org.owasp.mastestapp.MASTestApp-iOS -l ./script.js -l ../MASTG-DEMO-0044/script.js -o output.txt
```